dev-docs/module-rules.md

```diff
@@ -40,65 +40,77 @@ Bid adapters, user ID adapters, analytics adapters, and other modules will be re
 
 This set of module rules applies to both Prebid.js and Prebid Server.  However, these rules don’t necessarily apply to tools or the Prebid Mobile SDK. Rules and guidelines for those products will be determined by their respective committees as required.
 
-The use of the terms "must" and "should" in this document is deliberate.  However, some of the rules are "aspirational," in that we know there are existing exceptions to the rule that will need to get sorted out over time. These are marked with an asterisk (*). The goal is:
-
-- Every new module adheres to all required rules.
-- Even grandfathered modules must adhere to all required rules by Prebid.js 5.0.
+The use of the terms "must" and "should" in this document is deliberate.  However, some of the rules are "aspirational," in that we know there are existing exceptions to the rule that will need to get sorted out over time. These are marked with an asterisk (*).
 
 ### Global Module Rules
 
-1. Modules must(*) not load outside code files unless those libraries are approved and open source.
+#### Strictly Enforced Rules
+
+1. Each adapter and module must include working maintainer information with a group email address for Prebid.org to contact for support or maintenance.
```

Comment:
IMHO the wording "working maintainer" sounds a little confusing. How about omitting the word "working" and adding a second sentence along the lines of "The group email address must remain current and respond to enquiries in a timely manner."

Reply:
inquiries.